src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/README.md

```diff
@@ -0,0 +1,66 @@
+Default date picker:
```

Comment:
The `README` files are actually not only a set of examples, but should also explain stuff. What can be easily added here is e.g. what DatePicker is used with a link to it, and some explanations of the available properties (alongside with the examples). You can also have a look at the other `README.md` files for inspiration.

Reply:
I would just omit this sentence.